var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,9 +42,6 @@ class Relion(CMakePackage, CudaPackage):
                     'Profiling', 'Benchmarking'))
 
     depends_on('mpi')
-    # relion will not build with newer versions of cmake
-    # per https://github.com/3dem/relion/issues/380
-    depends_on('cmake@3:3.9.4', type='build')
```

Comment:
It looks like this should be:
```python
depends_on('cmake@2.8:', type='build')
```

Reply:
Adding back

@adamjstewart I reached out to Relion developer who claim 3.1 won't compile due to `-DTIMING` flag being added to cmake.  I don't have this in the template.  Are you familiar with that argument?

Reply:
Never mind @adamjstewart 

The Benchmarking build type seems to add `-DTIMING` which does not compile properly.  The developers will have to fix.  3.1 beta compiles without that build type.  I will send updates now.